README.md

```diff
@@ -310,8 +310,9 @@ msgs, err := conn.FetchMessages("<station-name>", "<consumer-name>",
 ```
 
 ### Fetch a single batch of messages after creating a consumer
+`prefetch = true` will prefetch next batch of messages and save it in memory for future Fetch() request
```

Comment:
please add a note for the user that he should set the max ack time big enough since the messages will sit in a local cache for some time before processing

Reply:
Added a note for the same.